curses.h

```diff
@@ -28,11 +28,6 @@ Defined by this header:
 **man-end****************************************************************/
 
 #define PDCURSES        1      /* PDCurses-only routines */
-#if defined( CHTYPE_32)
-   #define CHTYPE_LONG     1      /* chtypes will be 32 bits */
-#elif !defined( CHTYPE_16)
-   #define CHTYPE_LONG     2      /* chtypes will be (default) 64 bits */
-#endif
 
```

Comment:
Is the dropping of CHTYPE_16 (in constrast to support for CHTYPE_64) a change that @wmcbrine is fine to include or are there still environments where this would be reasonable to keep?
... and as I'm out of curses handling currently - what chtypes are used in ncurses?